config/detekt/baseline.xml

```diff
@@ -72,7 +72,6 @@
     <ID>LongMethod:BarChartViewHolder.kt$BarChartViewHolder$private fun BarChart.draw( item: BarChartItem, labelStart: TextView, labelEnd: TextView ): BarCount</ID>
     <ID>LongMethod:ClicksUseCase.kt$ClicksUseCase$override fun buildUiModel(domainModel: ClicksModel, uiState: SelectedClicksGroup): List&lt;BlockListItem></ID>
     <ID>LongMethod:HomepageSettingsDialog.kt$HomepageSettingsDialog$override fun onCreateDialog(savedInstanceState: Bundle?): Dialog</ID>
-    <ID>LongMethod:MediaPickerFragment.kt$MediaPickerFragment$override fun onViewCreated(view: View, savedInstanceState: Bundle?)</ID>
```

Comment:
Suggestion (💡): Now that you refactored this `onViewCreated(...)` method and correctly removed this `LongMethod` baseline warning, I noticed that you could also remove the `ComplexMethod` related such violation, see line `21` and `ComplexMethod:MediaPickerFragment.kt`. As such, with this PR, you are effectively fixing two baseline warnings at once, smooth! 🎉

Reply:
Fixed in: d88041f404d4d6b60529187876d16b1a50b17929